tests/inference/test_serve.py

```diff
@@ -20,20 +20,22 @@
 
 # Parametrize the test function with different combinations of parameters
 @pytest.mark.parametrize(
-    "config_file, models, port, simple, keep_serve_termimal",
+    "config_file, models, port, simple, keep_serve_termimal,list_model_ids",
```

Comment:
pay attention to the style
```suggestion
    "config_file, models, port, simple, keep_serve_termimal, list_model_ids",
```